jena-fuseki2/jena-fuseki-webapp/src/test/java/org/apache/jena/fuseki/TestAdmin.java

```diff
@@ -187,6 +189,21 @@
         addTestDataset(fileBase+"config-ds-plain-2.ttl");
         checkExists("test-ds2");
     }
+    
+    @Test public void add_delete_dataset_6() {
```

Comment:
Added a TDB2 based dataset so I could test `/$/compact/<dataset>`, since adding and deleting this dataset might be unstable on Windows I made this and other tests that use that dataset require a non-Windows OS